### DISCUSSION TITLE
Move asdf-yode here

### DISCUSSION THREAD

**Discussion by TheLastZombie** [asdf-community/.github#32] [General]
I created [asdf-yode](https://github.com/TheLastZombie/asdf-yode) since it wasn't available in any other package repository (APT, Brew, etc.). It's a fork of Node.js used mainly by/in conjunction with the [Yue](https://libyue.com/) library.

Let me know if I need to add someone as a collaborator or do anything else.

**Comment by smorimoto** [comment]
Consult this discussion post: https://github.com/orgs/asdf-community/discussions/57 This is the smoothest way to go. Of course I can do all the work for you, but it's done on a best effort basis. (I've been busy with work lately...)
